mcstatus/motd.py

```diff
@@ -0,0 +1,558 @@
+                    parsed_motd.append(Formatting.RESET)
+                else:
+                    parsed_motd.append(WebColor.from_hex(item["color"]))
+
+        for style_key, style_val in Formatting.__members__.items():
+            lowered_style_key = style_key.lower()
+            if item.get(lowered_style_key) is False or item.get(lowered_style_key) == "false":
+                try:
+                    parsed_motd.remove(style_val)
+                except ValueError:
+                    # some servers set the formatting keys to false here, even without it ever being set to true before
+                    continue
+            elif item.get(lowered_style_key) is not None:
+                parsed_motd.append(style_val)
+
+        # TODO Rewrite it to use `:=` after dropping support for Python 3.7
+        text = item.get("text")
+        if text is not None:
+            parsed_motd.extend(cls._parse_as_str(text, bedrock=bedrock))
+        translate = item.get("translate")
+        if text is None and translate is not None:
+            parsed_motd.append(TranslationTag(translate))
+        parsed_motd.append(Formatting.RESET)
+
+        if "extra" in item:
+            auto_add = list(filter(lambda e: type(e) is Formatting and e != Formatting.RESET, parsed_motd))
+
+            for element in item["extra"]:
+                parsed_motd.extend(cls._parse_as_dict(element, auto_add=auto_add.copy()))
+
+        return parsed_motd
+
+    @staticmethod
+    def _parse_as_str(raw: str, *, bedrock: bool = False) -> list[ParsedMotdItem]:
+        """Parse a MOTD when it's string.
+
+        .. note:: This method returns a lot of empty strings, use ``Motd.simplify`` to remove them.
+
+        :param raw: Raw MOTD, directly from server.
+        :param bedrock: Is server Bedrock Edition?
+            Ignores ``MinecraftColor.MINECOIN_GOLD`` if it's ``False``.
+        :returns: ``ParsedMotdItem`` list, which need to be passed to ``__init__``.
+        """
+        parsed_motd: list[ParsedMotdItem] = []
+
+        split_raw = MOTD_COLORS_RE.split(raw)
+        for element in split_raw:
+            clean_element = element.lstrip("&§").lower()
+            standardized_element = element.replace("&", "§").lower()
+
+            if standardized_element == "§g" and not bedrock:
+                parsed_motd.append(element)  # minecoin_gold on java server
+                continue
+
+            if standardized_element.startswith("§"):
+                try:
+                    parsed_motd.append(MinecraftColor(clean_element))
+                except ValueError:
+                    try:
+                        parsed_motd.append(Formatting(clean_element))
+                    except ValueError:
+                        # just a text
+                        parsed_motd.append(element)
+            else:
+                parsed_motd.append(element)
+
+        return parsed_motd
+
+    def simplify(self) -> Self:
```

Comment:
A name like `simplify` would imply that this is happening in-place, and is modifying the `Motd` instance, instead, let's call it `simplified`.

```suggestion
    def simplified(self) -> Self:
```

Reply:
Fixed [here](https://github.com/py-mine/mcstatus/compare/26690ce403010ab0af019cf2230d246e2920602e..ec2cacf3a1ee18ca0432a5c5e54c80ee6dbc3fcb), thanks!